### PR TITLE
:memo: SendGrid integration

### PR DESCRIPTION
**Pull request by vcampitelli**
Creating SendGrid SSO integration guide

@mooreds I did leave a bunch of _"@TODO screenshots"_ there. Can you replace the existing screenshots with yours? Or we could have #1989 merged instead to use the custom `safari` wrapper to add the image wireframe.

### CODE REVIEW

File: site/docs/v1/tech/samlv2/sendgrid.adoc [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+== Overview
+
+include::docs/v1/tech/samlv2/_saml_idp_note.adoc[]
+
+== Prerequisites
+
+This document assumes you have a running instance of FusionAuth and a working SendGrid application. You will also need admin accounts for both to configure them correctly.
+
+> Single Sign-On (SSO) is available for Twilio SendGrid Email API Pro, Premier, and Marketing Campaigns Advanced plans only. See the Twilio SendGrid pricing page for a full list of Twilio SendGrid features available by plan.
```
Comment (mooreds) [inline site/docs/v1/tech/samlv2/sendgrid.adoc:16]:
```suggestion
[NOTE.note]
====
Single Sign-On (SSO) is available for Twilio SendGrid Email API Pro, Premier, and Marketing Campaigns Advanced plans only. See the Twilio SendGrid pricing page for a full list of Twilio SendGrid features available by plan.
====
```

File: site/docs/v1/tech/samlv2/sendgrid.adoc [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+layout: doc
+title: SAML v2 & SendGrid
+description: Setting up SendGrid to login using FusionAuth via SAML v2
+navcategory: login-methods
+---
+
+== Overview
+
+include::docs/v1/tech/samlv2/_saml_idp_note.adoc[]
+
```
Comment (mooreds) [inline site/docs/v1/tech/samlv2/sendgrid.adoc:11]:
```suggestion
[NOTE.note]
====
This document helps you set up single sign-on using SAML for your SendGrid account. If you are looking to set up SendGrid as an email provider, you want link:/docs/v1/tech/email-templates/configure-email#sendgrid[this documentation instead].
====

```

**Comment by vcampitelli** [comment]
I've added a step explaining how to set the Tenant issuer.

**Review by mooreds — APPROVED** [review]
Ship it!

**Pull request merged**